From 5872557a54f440d8cf046714508898b173885399 Mon Sep 17 00:00:00 2001
From: Yong Wang <yong.y.wang@intel.com>
Date: Fri, 5 Feb 2010 11:19:36 +0800
Subject: [PATCH] linux-2.6.29-kms-after-sata.patch

This patch is from Arjan ven de Ven <arjan@linux.intel.com>

Signed-off-by: Yong Wang <yong.y.wang@intel.com>
---
 drivers/Makefile |   15 ++++++++-------
 1 files changed, 8 insertions(+), 7 deletions(-)

diff --git a/drivers/Makefile b/drivers/Makefile
index 6ee53c7..5dc6dd3 100644
--- a/drivers/Makefile
+++ b/drivers/Makefile
@@ -26,15 +26,8 @@ obj-$(CONFIG_REGULATOR)		+= regulator/
 # default.
 obj-y				+= char/
 
-# gpu/ comes after char for AGP vs DRM startup
-obj-y				+= gpu/
-
 obj-$(CONFIG_CONNECTOR)		+= connector/
 
-# i810fb and intelfb depend on char/agp/
-obj-$(CONFIG_FB_I810)           += video/i810/
-obj-$(CONFIG_FB_INTEL)          += video/intelfb/
-
 obj-y				+= serial/
 obj-$(CONFIG_PARPORT)		+= parport/
 obj-y				+= base/ block/ misc/ mfd/
@@ -46,6 +39,14 @@ obj-$(CONFIG_ATA)		+= ata/
 obj-$(CONFIG_MTD)		+= mtd/
 obj-$(CONFIG_SPI)		+= spi/
 obj-y				+= net/
+
+# gpu/ comes after char for AGP vs DRM startup
+obj-y				+= gpu/
+
+# i810fb and intelfb depend on char/agp/
+obj-$(CONFIG_FB_I810)           += video/i810/
+obj-$(CONFIG_FB_INTEL)          += video/intelfb/
+
 obj-$(CONFIG_ATM)		+= atm/
 obj-$(CONFIG_FUSION)		+= message/
 obj-$(CONFIG_FIREWIRE)		+= firewire/
-- 
1.5.5.1

